src/app/page.jsx

```diff
@@ -0,0 +1,22 @@
+'use client'
+
+import Message from '../components/message'
```

Comment:
```suggestion
import Message from '@/components/message'
```

Reply:
Use [Absolute Imports and Module Path Aliases](https://nextjs.org/docs/app/building-your-application/configuring/absolute-imports-and-module-aliases). Later when you will have nested directories relative paths will look ugly.

Reply:
Ok, will change it